### DISCUSSION TITLE
Maiking Gifts

### DISCUSSION THREAD

**Discussion by Buul135** [Monika-After-Story/MonikaModDev#10249] [QA - code and content creation (technical)] [answered]
I couldn't find anything that tell's me how to make custom gifts, unless I'm missing something

**Comment by RZed01** [comment]
That would be on this Github's wiki at https://github.com/Monika-After-Story/MonikaModDev/wiki/Gift-Giving-Guide

**Comment by Buul135** [comment]
> That would be on this Github's wiki at https://github.com/Monika-After-Story/MonikaModDev/wiki/Gift-Giving-Guide

I was looking for how to make custom gifts, not how to gift ones already in the game

**Comment by Booplicate** [comment]
What kind of gift?

**Comment by Buul135** [comment]
> What kind of gift?

I thought it would be nice if I could gift her things like books, non-sprite stuff if thats possable

**Comment by Booplicate** [comment, marked as answer]
It's not as well support since your custom gifts may interfere the builtin gifts (or those that will be added in the future), but generally look at how our gifts work, it's not hard to define more: https://github.com/Monika-After-Story/MonikaModDev/blob/06baf319a34c2ef585bc7c0a1e969a7eaa894b35/Monika%20After%20Story/game/zz_reactions.rpy#L1608-L1623

**Reply by Buul135** [reply]
thanks that work, I should have thought to look at the other gift files
